Tools/PIDML/CMakeLists.txt

```diff
@@ -36,3 +36,9 @@ o2physics_add_dpl_workflow(qa-pid-ml
                   SOURCES qaPidML.cxx
                   PUBLIC_LINK_LIBRARIES O2::Framework O2Physics::AnalysisCore ONNXRuntime::ONNXRuntime
                   COMPONENT_NAME Analysis)
+
+o2physics_add_dpl_workflow(KaonPidTask
```

Comment:
I indeed overlooked this - you should name your executable `kaon-pid`, for example

Reply:
Ok done